project_name/config.py

```diff
@@ -0,0 +1,47 @@
+import base64
+import os
+from typing import List
+
+from goodconf import GoodConf, Field
+from pathlib import Path
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "{{ project_name }}.settings")
+
+
+class AppConfig(GoodConf):
+    """Configuration for {{ project_name }}"""
+
+    DEBUG: bool
+    ALLOWED_HOSTS: List[str] = Field(
+        ["*"],
+        initial=lambda: ["*"],
+        description="Hosts allowed to serve the site "
+        "https://docs.djangoproject.com/en/{{ docs_version }}/ref/settings/#allowed-hosts",
+    )
+    DATABASE_URL: str = Field(
+        "sqlite:///PATH",
+        initial=lambda: "sqlite:///PATH",
```

Comment:
For `DATABASE_URL`, I'd set the default to either `sqlite:///./sqlite3.db` or `f"sqlite:///{PROJECT_DIR}/sqlite3.db`. No need for an `initial` in that case.